config/default/migrate_plus.migration.hs_capx_images.yml

```diff
@@ -0,0 +1,88 @@
+uuid: c97e6ea0-2d0e-4eae-99b0-6d584c7ce942
+langcode: en
+status: true
+dependencies: {  }
+id: hs_capx_images
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags: {  }
+migration_group: hs_capx
+label: 'H&S CAPx Image Importer'
+source:
+  constants:
+    status: 1
+    type: image
+    file_destination: 'public://media/capx/'
+    file_extension: .jpg
+    separator: '-'
+  urls: {  }
+  item_selector: values
+  fields:
+    -
+      name: sunetid
+      label: sunetid
+      selector: uid
+    -
+      name: display_name
+      label: 'Display Name'
+      selector: displayName
+    -
+      name: alias
+      label: Alias
+      selector: alias
+    -
+      name: profile_photo
+      label: 'Profile Photo'
+      selector: profilePhotos/bigger/url
+    -
+      name: etag
+      label: 'CAP API Etag'
+      selector: meta/etag
+  ids:
+    sunetid:
+      type: string
+process:
+  name: display_name
+  status: constants/status
+  bundle: constants/type
+  image_timestamp:
```

Comment:
There is a `LastModified` key on the response for each image. Why not use that?

Reply:
That rings a bell. Where did that conversation happen? Slack, github?

Reply:
it was in slack. https://stanfordwebservices.slack.com/archives/D2YMF9509/p1509739760000136
>>> Mike: The interesting thing now, we don’t even have to apply those changes in that PR. the image urls now contain a timestamp 
`https://cap.stanford.edu/profiles/viewImage?profileId=162949&type=big&ts=1509739817173`

Shea McKinney [1:25 PM]
Huh interesting. The path to the image finally changes
So yeah, we don't need that timestamp check
anyway